src/util/account/accountUtils.ts

```diff
@@ -1,41 +1,31 @@
-import algosdk, {Algodv2, IntDecoding} from "algosdk";
+import algosdk, {Algodv2} from "algosdk";
 import {fromByteArray, toByteArray} from "base64-js";
 
 import {getContract} from "../../contract";
 import {ContractVersionValue} from "../../contract/types";
 import {
-  BASE_MINIMUM_BALANCE,
   MINIMUM_BALANCE_REQUIRED_PER_APP,
   MINIMUM_BALANCE_REQUIRED_PER_ASSET,
   MINIMUM_BALANCE_REQUIRED_PER_BYTE_SCHEMA,
-  MINIMUM_BALANCE_REQUIRED_PER_CREATED_APP,
-  MINIMUM_BALANCE_REQUIRED_PER_EXTRA_APP_PAGE,
   MINIMUM_BALANCE_REQUIRED_PER_INT_SCHEMA_VALUE
 } from "../constant";
 import {V1PoolInfo} from "../pool/poolTypes";
 import {decodeState, encodeString, joinByteArrays} from "../util";
 import {
   AccountExcess,
   AccountExcessWithinPool,
-  AccountInformation,
   AccountInformationData
 } from "./accountTypes";
 
-export function getAccountInformation(
-  client: Algodv2,
-  address: string,
-  intDecoding: IntDecoding = IntDecoding.DEFAULT
-) {
+export function getAccountInformation(client: Algodv2, address: string) {
   return new Promise<AccountInformationData>(async (resolve, reject) => {
     try {
-      const accountInfo = await (client
-        .accountInformation(address)
-        .setIntDecoding(intDecoding)
-        .do() as Promise<AccountInformation>);
+      const accountInfo = await client.accountInformation(address).do();
 
       resolve({
         ...accountInfo,
-        minimum_required_balance: calculateAccountMinimumRequiredBalance(accountInfo)
+        assets: accountInfo.assets ?? [],
+        appsLocalState: accountInfo.appsLocalState ?? []
```

Comment:
After the changes, this function is basically just returning `accountInfo` which is the result of `await client.accountInformation(address).do()`
So it's not doing any useful abstraction, I think we can remove it, also the `AccountInformationData` custom type.

In the codebase, we can replace:
`getAccountInformation()` -> `client.accountInformation(...).do()`
`AccountInformationData` -> `modelsv2.Account`

Minimizing the number of custom types/functions simplifies the codebase. Also it's a good time since this will be a breaking change anyway.